lib/jgi_mg_assembly/utils/report.py

```diff
@@ -143,17 +213,41 @@ def _write_html_file(self, html_file_name, stats_files, reads_info):
         with open(html_file_name, "w") as outfile:
             outfile.write(html_content)
 
-    def _protocol_text(self):
+    def _protocol_text(self, pipeline_output):
+        # if rqcfilter is run, its command looks like this:
+        #
+        # and if not, it looks like this:
+        # "BBTools.run_RQCFilter_local -- skipped. No command run."
+        rqcfilter_version = pipeline_output["rqcfilter"]["version_string"]
```

Comment:
I don't see this getting used

Reply:
Oops, forgot the optional RQCFilter section of the report text. Added.